notebooks/122-quantizing-model-with-accuracy-control/README.md

```diff
@@ -0,0 +1,26 @@
+# Quantization with accuracy control using NNCF
+
+This tutorial demonstrates how to apply `INT8` quantization with accuracy control to the speech recognition model and to PyTorch YOLOv8 model.
```

Comment:
I think it is two tutorials. So, it is better to rephrase it like `These tutorials demonstrate how to apply 8-bit quantization with accuracy control to the speech recognition and YOLOv8 models:` and then continue with the list.
I also wonder why it is two-in-one at the same time.

Reply:
Rephrased